src/Parser/BrewmasterMonk/Constants.js

```diff
@@ -26,3 +26,22 @@ export const SPELLS_WHICH_REMOVE_BOC = [
   SPELLS.IRONSKIN_BREW.id,
   SPELLS.PURIFYING_BREW.id,
 ];
+
+export const SPELLS_WITH_ENERGY_COST = {
```

Comment:
Specifying the `energyCost` in the spells object is preferred (and already available that way for talents), see `manaCost` for examples.